File: Week_4/problem_1-Shape_1.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219432/problem/P

#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int t = n;

    for (int i = 1; i <= n; i++){
        for (int j = 1; j <= t; j++){
            cout << "*";
        }
        t--;
        cout << endl;
    }

    return 0;
}

File: Week_4/problem_1-Shape_1.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219432/problem/P

n = int(input())
t = n

for i in range(n):
    for j in range(t):
        print("*", end="")  # Use `end=""` to print stars on the same line
    t -= 1
    print()  # Move to the next line after printing all stars for the current row
